charts/alfresco-ai-transformer/README.md

```diff
@@ -0,0 +1,55 @@
+# alfresco-ai-transformer
+
+![Version: 0.1.0-alpha.0](https://img.shields.io/badge/Version-0.1.0--alpha.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 1.4.5](https://img.shields.io/badge/AppVersion-1.4.5-informational?style=flat-square)
+
+A Helm chart for deploying Alfresco ai transformer service
+
+## Requirements
+
+| Repository | Name | Version |
+|------------|------|---------|
+| https://alfresco.github.io/alfresco-helm-charts/ | activemq | 3.0.1 |
+| https://alfresco.github.io/alfresco-helm-charts/ | alfresco-common | 2.0.0 |
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| activemq.enabled | bool | `false` |  |
+| aws | object | `{"accessKey":"XXXXXXXXXXXXXXXXXXXXXXXX","comprehendRoleARN":"arn:aws:iam::XXXXXXXXXXXX:role/ComprehendAsyncJobs","region":"region-name","s3Bucket":"s3-bucket-name","secretAccessKey":"XXXXXXXXXXXXXXXXXXXXXXXX"}` | To configure these, you need to provide or export your AWS credentials https://docs.alfresco.com/intelligence-services/latest/config/#default-configuration |
```

Comment:
I forgot that helm-docs render non-leaf documentation really bad

move it on top of `accessKey`?